data/regions/RO.yml

```diff
@@ -14,7 +14,6 @@ zip_regex: "^(RO-?)?\\d{6}$"
 zip_example: '060274'
 phone_number_prefix: 40
 building_number_required: true
-zip_requirement: recommended
```

Comment:
A region's zip is implicitly required if there is no `zip_requirement` attribute in the country config but a `zip_regex` attribute is defined.